packages/core/src/components/pane/pane.scss

```diff
@@ -0,0 +1,217 @@
+  }
+}
+
+:host {
+  @include ix-component;
+  background-color: var(--theme-color-2);
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  box-sizing: border-box;
+
+  aside {
+    width: 100%;
+    height: 100%;
+  }
+
+  .title-text {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    gap: var(--theme-space-1);
+
+    .title-text-overflow {
+      position: relative;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    .title-text-left-right-expanded {
+      order: 1;
+    }
+  }
+
+  .header-gap {
+    gap: var(--theme-space-1);
+  }
+
+  .top-bottom-pane {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+
+    .title {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      margin: var(--theme-space--1) var(--theme-space-2) var(--theme-space--1) var(--theme-space-2);
+    }
+
+    .title-finished {
+      padding: 0 !important;
+      margin: var(--theme-space-2) var(--theme-space-2) var(--theme-space-2) var(--theme-space-2);
+    }
+
+    .title-expanded {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .title-hide-on-collapse {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: space-between;
+      padding: 0 !important;
+      margin: var(--theme-space-2) var(--theme-space-2) var(--theme-space-2) var(--theme-space-2);
+    }
+  }
+
+  .left-right-pane {
+    display: flex;
+    flex-grow: 1;
+
+    .title {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      margin: var(--theme-space-2) var(--theme-space--1)  var(--theme-space-2) var(--theme-space--1);
+
+      .title-text {
+        flex-direction: row-reverse;
+        justify-content: start;
+      }
+    }
+
+    .title-finished {
+      padding: 0 !important;
+      margin: var(--theme-space-2) var(--theme-space-2) var(--theme-space-2) var(--theme-space-2);
+    }
+
+    .title-expanded {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .title-hide-on-collapse {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: space-between;
+      padding: 0 !important;
+      margin: var(--theme-space-2) var(--theme-space-2) var(--theme-space-2) var(--theme-space-2);
+    }
+  }
+
+  .left-right-pane.expanded {
+    flex-direction: column;
+  }
+
+  .rotate {
+    writing-mode: vertical-lr;
+    text-orientation: mixed;
+    transform: rotate(180deg);
+  }
+
+  .mobile-pane {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+
+    .title-mobile {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: space-between;
+      padding: 0 !important;
+      margin: var(--theme-space-1) var(--theme-space-2);
+    }
+  }
+
+  .side-pane-content {
+    padding: var(--theme-space-1) var(--theme-space-2);
+    overflow: auto;
```

Comment:
It looks as if the content overflow will not work in all cases. The scrollbar does not appear if there is only a slight overflow (e.g. ~ 100px). For lager content it works.